### DISCUSSION TITLE
Authority Matrix's Impact on Dashboard and Automations

### DISCUSSION THREAD

**Discussion by zonistefano** [teableio/teable#1114] [Q&A]
Hi!
First, I want to express my gratitude for the incredible work you have done with Teable, it's an amazing tool. 

I recently came across some documentation [here](https://help.teable.cn/ji-ben-gong-neng/quan-xian-guan-li) which showcases screenshots of the authority matrix. 
However, it appears that those images primarily illustrate how to control access to tables. Could you clarify whether the authority matrix also influences access controls on the dashboard and automations? 
I think that being able to manage these aspects within the authority matrix is particularly important, for example it allows teams to be self-accountable for their areas without granting excessively broad permissions, maintaining both flexibility and security.

Thank you once again for your dedication and for creating such a valuable tool. I look forward to your insights!

**Comment by tea-artist** [comment]
Thanks for your support! 
Right now, automation only allows base admin access after enabled authority matrix. The dashboard will be included in this management soon.
